ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingProcess.java

```diff
@@ -72,15 +45,13 @@ public ContinuousHikingProcess(DRCRobotModel robotModel)
                                                                             syncedRobot.getReferenceFrames(),
                                                                             controllerFootstepQueueMonitor,
                                                                             continuousHikingLogger,
-                                                                            continuousHikingParameters,
-                                                                            monteCarloPlannerParameters,
-                                                                            footstepPlannerParameters,
-                                                                            swingPlannerParameters);
+                                                                            activeMappingParameterToolBox);
 
       Runtime.getRuntime().addShutdownHook(new Thread(this::destroy, "Shutdown"));
 
       // Add initial delay to get things going in the right order
-      executorService.scheduleAtFixedRate(this::update, 8500, 100, TimeUnit.MILLISECONDS);
+      ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
```

Comment:
We should use a ThreadFactory so the threads are identifiable in profilers. Otherwise they come up something like Thread-1, Thread-2, etc.
```suggestion
      ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1, ThreadTools.createNamedThreadFactory(getClass().getSimpleName() + "Update"));
```

Reply:
I created a `ThreadFactoryBuilder` from `com.google.common.util.concurrent.ThreadFactoryBuilder;`